.github/workflows/lint-test.yaml

```
@@ -24,18 +24,17 @@ jobs:
         run: cat $TEST_VALUES_PATH.tpl | envsubst > $TEST_VALUES_PATH
 
       - name: Set up Helm
-        uses: azure/setup-helm@v1
+        uses: azure/setup-helm@v3
         with:
-          version: v3.8.1
+          version: v3.11.2
```

https://github.com/helm/helm/releases/tag/v3.12.1

```suggestion
          version: v3.12.1
```